.github/workflows/test.yml

```diff
@@ -37,4 +29,4 @@ jobs:
         with:
           java-version: "openjdk@1.11"
       - name: Tests
-        run: sbt test it:test
+        run: sbt +test it:test
```

Comment:
as flink with scala 2.13 requires custom image I don't do container based tests - the same as in the main repo as of now